app/controllers/food_controller.rb

```diff
@@ -0,0 +1,12 @@
+class FoodController < ApplicationController
+  def show
+    @query = params[:q]
+
+    conn = Faraday.new(url: "https://api.nal.usda.gov") do |faraday|
+        faraday.params[:api_key] = ENV['food_api_key']
+    end
+    food_response = conn.get("/fdc/v1/foods/search?query=#{@query}")
+    @foods = JSON.parse(food_response.body, symbolize_names: true)
```

Comment:
Ideally, I'd like to see this refactored to a service, facade pattern with poros. I know you lost a bit of time up front so you weren't able to follow red, green, refactor, but this is the first thing I'd probably refactor after adding some tests when you have time.